src/api/app/controllers/webui/request_controller.rb

```diff
@@ -209,12 +209,7 @@ def accept_request
 
   def forward_request_to(fwd)
     tgt_prj, tgt_pkg = params[fwd].split('_#_') # split off 'forward_' and split into project and package
-    description = @req.description
-    description ||= ""
-    who = @req.creator
-    description += ' (forwarded request %d from %s)' % [params[:number], who]
 
-    req = nil
```

Comment:
It will fail anyway. Does it make any difference if it fails because `undefined local variable or method 'req'` or `undefined method 'number' for nil:NilClass`?